### ISSUE TITLE
[BUG] Inconsistent dependency on ArnoldiMethod

### ISSUE THREAD

**Issue by gkantsidis** [JuliaGraphs/Graphs.jl#354]
The [Project.toml](https://github.com/JuliaGraphs/Graphs.jl/blob/master/Project.toml) for the project in v.1.9 restricts ArnoldiMethod to v0.3 (`ArnoldiMethod = "0.3"`). However, in the official package repository (https://juliahub.com/ui/Packages/General/Graphs), the compatibility constraint is for v1.0 or v2.0 (ArnoldiMethod = "0.1, 0.2"). 
As a result, using the package still restricts ArnoldiMethod to v0.1 or v0.2.

It seems that the dependency has been updated but without updating this packages version number and without pushing the change to the official repo.

Would it be possible to fix this please?

**Comment by gdalle** [comment]
The PR fixing it has been waiting for a review, if you review it I can merge and tag a release

https://github.com/JuliaGraphs/Graphs.jl/pull/344

**Comment by gkantsidis** [comment]
The current (from #344) version again seems to fix the version of ArnoldiMethod.jl (thanks!), but we will also need to increase the version number of this package (e.g. to 1.9.1) (otherwise, there will still be problems).

On the use of [Arn](https://github.com/JuliaLinearAlgebra/ArnoldiMethod.jl):
The comments of the commit: 
https://github.com/JuliaLinearAlgebra/ArnoldiMethod.jl/commit/56221508b32e2be8cc041733f32f1fd669ba1490
seems to suggest (to me) that the authors would like to discourage the use of LR(), LM() and the like, and encourage the use of :LR, :LM, etc.
I have created a pull request in https://github.com/JuliaGraphs/Graphs.jl/pull/355 that attempts to fix that.

**Comment by gdalle** [comment]
Yeah I know we need to tag a new release, I'm just waiting to merge one or two more PRs first. This will happen next week at the latest

**Comment by simonschoelly** [comment]
Do you need a new release urgently? Otherwise you also specify that you want use `add Graphs#master` to use the latest master branch - there is probably also a way to specify a commit hash - but I did not find the doc on that atm.

**Comment by gkantsidis** [comment]
Thanks. I do need a new release (an internal project has a direct dependency on ArnoldiMethod.jl, and a transitive dependency on Graphs -- so, it does not seem possible to just `add Graphs#master`). However, we can wait a few days for @gdalle to create the new release. 
Thanks to both!

**Comment by gdalle** [comment]
@simonschoelly there's no problem with releasing often, right? Since we have adopted ColPrac, the standard could even be to release after each PR

**Comment by thchr** [comment]
This is resolved with the package registration from last week, right?
